ELSpace/Screens/Hub/Activities/ViewModels/ActivitiesViewModel.swift

```diff
@@ -5,7 +5,7 @@ import SwiftDate
 protocol ActivitiesViewModelProtocol {
     var dataSource: Observable<[DailyReportViewModelProtocol]> { get }
     var isLoading: Observable<Bool> { get }
-    var month: String { get }
+    var monthObservable: Observable<String> { get }
```

Comment:
Is there any reason why it has to have `Observable` in the name itself? I think just `month` should be sufficient.

Reply:
Changing back to month 👍